adapters/unicredit-adapter/src/test/java/de/adorsys/xs2a/adapter/unicredit/UnicreditPaymentInitiationServiceTest.java

```diff
@@ -139,6 +141,35 @@ void updatePaymentPsuData() {
         assertThat(requestBuilder.headers()).containsEntry(RequestHeaders.PSU_ID_TYPE, DEFAULT_PSU_ID_TYPE);
     }
 
+    @Test
+    void updatePaymentPsuData_responseMapping() {
+        Request.Builder requestBuilder = spy(new RequestBuilderImpl(httpClient, "PUT", AUTHORISATION_URL));
+
+        UnicreditUpdatePsuAuthenticationResponse uniCreditResponse = new UnicreditUpdatePsuAuthenticationResponse();
```

Comment:
`uniCreditResponse` - looks like a typo, should be `unicreditResponse`

Reply:
fixed and squashed